### DISCUSSION TITLE
how to disable infoLog Possible templates for resolved url: ... on the production?

### DISCUSSION THREAD

**Discussion by bebjakub** [wpengine/faustjs#1451] [Q&A]
Hello,
how to disable info log [Possible templates for resolved url: ...](https://github.com/wpengine/faustjs/blob/main/packages/faustwp-core/src/getWordPressProps.tsx#L103), pls?

On the production I think it should be disabled, It can generate quite large unnecessary logs.

Thank you!

**Comment by ccorda** [comment]
As Vercel is changing pricing around log drains, we're looking at our log usage, and this is responsible for a lot of production logging. Would it be possible to make this disabled in production by default, or with something in faust config?
